test/format/config.h

```diff
@@ -334,6 +334,10 @@ static CONFIG c[] = {
 	  "maximum time to run in minutes",
 	  C_IGNORE, 0, 0, UINT_MAX, &g.c_timer, NULL },
 
+    { "timing_stress_aggressive_sweep",
```

Comment:
The indentation and white spaces do not follow the rest of the file. Take another look and see if you can fix these.